slack-api-client/src/test/java/test_locally/api/methods/SlackApiRequestTest.java

```diff
@@ -0,0 +1,58 @@
+package test_locally.api.methods;
+
+import com.slack.api.Slack;
+import com.slack.api.SlackConfig;
+import com.slack.api.methods.SlackApiRequest;
+import com.slack.api.methods.request.auth.AuthTestRequest;
+import com.slack.api.methods.response.auth.AuthTestResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import util.MockSlackApiServer;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static util.MockSlackApi.ExpiredToken;
+import static util.MockSlackApi.ValidToken;
+
+public class SlackApiRequestTest {
```

Comment:
Please add a test case to verify that `SlackApiRequest.setToken()` throws the correct exception.

Reply:
Added